### PR TITLE
Web component's broken tooltips fixes

### PR DESCRIPTION
**Pull request by preetvadaliya**
This PR fix issue #2445.

### CODE REVIEW

**Comment by AppInventorWorkerBee** [comment]
Can one of the admins verify this patch?

**Comment by preetvadaliya** [comment]
@ewpatton sir,

After the changes, I compared `simple_components.json` for both versions of `ComponentProcessor.java`, but there are no additional changes, instead of the tooltip changes to the "DataChanged" event.

File: appinventor/components/src/com/google/appinventor/components/runtime/CloudDB.java [hunk 1]
```diff
@@ -1111,7 +1111,8 @@ public void TagList(List<String> value) {
    * @param tag the tag that has changed.
    * @param value the new value of the tag.
    */
-  @SimpleEvent
+  @SimpleEvent(description = "Indicates that the data in the CloudDB project has changed. Launches an event with the " +
+      "`tag` that has been updated and the `value` it now has.")
```
Comment (SusanRatiLane) [inline appinventor/components/src/com/google/appinventor/components/runtime/CloudDB.java:1115]: This wording might lead people think that this event launches another event.

I'd recommend: "Event indicating that CloudDB data has changed for the given tag and value."

File: appinventor/components/src/com/google/appinventor/components/scripts/ComponentProcessor.java [hunk 2]
```diff
@@ -423,7 +423,7 @@ public void setDescription(String description) {
       } else {
         // Throw out the first @ or { and everything after it,
         // in order to strip out @param, @author, {@link ...}, etc.
-        this.description = description.split("[@{]")[0].trim();
+        this.description = description.split("[@|{@]+(([a-zA-Z])*)+[(\\s)*]")[0].trim();
```
Comment (SusanRatiLane) [inline appinventor/components/src/com/google/appinventor/components/scripts/ComponentProcessor.java:426]: This seems to test correctly. I'd like to ask another dev to take a look as a sanity check.

**Review by SusanRatiLane — CHANGES_REQUESTED** [review]
Small text change.

**Comment by SusanRatiLane** [comment]
@AppInventorWorkerBee  ok to test

**Comment by SusanRatiLane** [comment]
@AppInventorWorkerBee ok to test.

**Comment by ewpatton** [comment]
@AppInventorWorkerBee retest this please

**Review by SusanRatiLane — APPROVED** [review]

**Pull request merged**